dolfyn/turbulence.py

```diff
@@ -1,53 +0,0 @@
-import numpy as np
```

Comment:
Where did this code end up?

Reply:
dolfyn/adv/turbulence.py
I renamed TurbBinner to ADVBinner because all the turbulence functions were ADV related. Idea would be to eventually have one for ADCP data